packages/ui/src/molecules/QuantitySelector/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './QuantitySelector'
+export type { QuantitySelectorProps } from './QuantitySelector'
```

Comment:
The export is missing in the `[...]/src/index.ts` file.

Reply:
Thanks! Fixed it.